hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -568,7 +568,11 @@ public int getRange() {
     JUPYTER_SERVER_ALREADY_RUNNING(43, "Jupyter Notebook Server is already running", Response.Status.BAD_REQUEST),
     ERROR_EXECUTING_REMOTE_COMMAND(44, "Error executing command over SSH", Response.Status.INTERNAL_SERVER_ERROR),
     OPERATION_NOT_SUPPORTED(45, "Supplied operation is not supported", Response.Status.BAD_REQUEST),
-    GIT_COMMAND_FAILURE(46, "Git command failed to execute", Response.Status.BAD_REQUEST);
+    GIT_COMMAND_FAILURE(46, "Git command failed to execute", Response.Status.BAD_REQUEST),
+    ELASTIC_QUERY_NO_MAPPING(47, "Elastic query uses a field that is not in the mapping of the index",
+      Response.Status.BAD_REQUEST),
```

Comment:
Should there not also be an error code for "Elastic Service not available"?

Reply:
Theo and Mahmoud decided we will abort this PR and Mahmoud will create a new PR when Mahmoud's refactor and update of the elasticsearch client is ready.